apps/editor/widget/modal/create.jsx

```diff
@@ -113,9 +111,11 @@ const socialDbAdapter = {
       force: "true",
       onCommit: (v) => {
         console.log(v);
+        onOpenChange();
       },
       onCancel: (v) => {
         console.log(v);
+        onStoreSavedEmbedPath(lastPath)
```

Comment:
What's the thought behind this? lastPath vs path?

Reply:
lastPath is the last saved post path. This is used to pre-populate the post embed modal. What if the user click on save a new document but cancel it? We'd like to keep using the last saved path.